neo/Storage/Interface/DBInterface.py

```diff
@@ -2,11 +2,37 @@
 from neo.SmartContract.Iterable import EnumeratorBase
 from neo.logging import log_manager
 
-logger = log_manager.getLogger('db')
+logger = log_manager.getLogger()
 
 
-class DBCollection:
-    DB = None
+class DBProperties:
+
+    """ Store the different DB properties
+    Used to pass the relevant information to
+    neo.Storage.Implementation.[BACKEND].[BACKEND]DBClassMethods.openIter
```

Comment:
> Note that I used the include_value/key description directly from the Plyvel documentation for Iterators because I think that's the generic idea.

Good idea, that's exactly what it's used for. 

> The suggested descriptions describes the effect it has and requires no reasoning of the reader.

Yes, for me it was clear what it does, but thinking about it as a person from the other side of the fence, it makes totally sense.